Automattic/wp-calypso
# G Suite: Update Thank You page and nudge for Google Workspace

#### Changes proposed in this Pull Request

* update the checkout thank you page to use the proper product name
* update the checkout gsuite nudge to use the active google mail service name

#### Testing instructions

* You can open an existing G Suite thank you page with `/checkout/thank-you/{domain}/{receiptID}` to verify how that looks
* I'm not sure how to test the nudge as I don't think it's referenced anywhere
